requirements-test.in

```diff
@@ -5,4 +5,4 @@ importlib-resources==5.4.0
 jsonschema
 pytest
 pytest-cov
-pyyaml
+pyyaml
```

Comment:
unnecessary change

Reply:
I still see this change here, please keep POSIX endlines